Formula/u/umka.rb

```diff
@@ -0,0 +1,27 @@
+class Umka < Formula
```

Comment:
I feel like umka-lang would be a better name
```suggestion
class UmkaLang < Formula
```

Reply:
I think it is a bit confusing when package and executable have different names (on the other hand some formulae do that, if you really think it would be better then no problems)

Reply:
I know it is debatable, but I think umka-lang is better suited.

as for formula-name and artifact-name discrepancy, we have lots of them in the core project, sometimes we add alias to mitigate it .